src/test/addresses_deployers.sol

```diff
@@ -32,7 +32,8 @@ contract Deployers {
             0x4D6fbF888c374D7964D56144dE0C0cFBd49750D3,  // Oracles
             0x1f42e41A34B71606FcC60b4e624243b365D99745,  // Oracles
             0x075da589886BA445d7c7e81c472059dE7AE65250,  // Used for Optimism & Arbitrum bridge contracts
-            0x7f06941997C7778E7B734fE55f7353f554B06d7d   // Starknet
+            0x7f06941997C7778E7B734fE55f7353f554B06d7d,  // Starknet
+            0xb27B6fa77D7FBf3C1BD34B0f7DA59b39D3DB0f7e
```

Comment:
Add a comment here that new address is CES

Reply:
Fixed

Reply:
Resolving this